### PR TITLE
Changed definition of two tables, and added more debugging statements.

### PR DESCRIPTION
**Pull request by baslerj**

### CODE REVIEW

**Review by nkrs44 — CHANGES_REQUESTED** [review]
Only small edits in Definition.py, you should be able to change this within a few seconds. 
Please wait with merging for my research on tag limit. 
Also I'm not sure if changing the `id` from fields to tags causes a required change in the continuous queries, checking this myself.

**Comment by nkrs44** [comment]
I checked, it is required to also adjust the continuous queries to avoid a grouping via the vm's `id`.
Luckly i have already a method implemented, but it's now some "unnecessary" or annoying code.
For now, we group by `* `, therefore all tags. To not group by the `id` we need to specify the others individually.

I will push the code by myself later on, otherwise feel free to copy-edit-paste it from the `vadps` table on line 674.

File: python/influx/influx_client.py [hunk 1]
```diff
@@ -409,16 +410,19 @@ def insert_dicts_to_buffer(self, table_name: str, list_with_dicts: List[Dict[str
...
         query_buffer = []
+        appendCount = 0
         for mydict in list_with_dicts:
             try:
                 # split dict according to default tables
                 (tags, values, timestamp) = table.split_by_table_def(mydict=mydict)
 
                 if(isinstance(timestamp, str)):
                     timestamp = int(timestamp)
+                # LOGGER.debug("%d %s %s %d",appendCount,tags,values,timestamp)
```
[the remark this answers is not in the document]
Comment (nkrs44) [reply python/influx/influx_client.py:421]: I left it in the code since it is commented.

**Review by nkrs44 — APPROVED** [review]
@baslerj Please review my changes, i wasnt able to run the code due my system.
We still have the issue discussed in #6 , but the code is otherwise OK.

**Review by baslerj — COMMENTED** [review]
Changes look good.

File: python/influx/influx_client.py [hunk 2]
```diff
@@ -430,6 +434,8 @@ def insert_dicts_to_buffer(self, table_name: str, list_with_dicts: List[Dict[str
         # safeguard to avoid memoryError
         if(len(self.__insert_buffer[table]) > 5 * self.__query_max_batch_size):
             self.flush_insert_buffer()
+        LOGGER.debug("Appended %d items to the insert buffer", appendCount)
```
[the remark this answers is not in the document]
Comment (baslerj) [reply python/influx/influx_client.py]: Looks good, thanks

**Pull request merged**